prac_06/guitars.py

```diff
@@ -0,0 +1,37 @@
+"""
+CP1404/CP5632 Practical
+Guitar program that uses the Guitar class
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Guitar program."""
```

Comment:
You could add a little more description to this docstring, just what does this guitar program do, such as:
"""Program take gets guitar information and prints formatted list""" or something

Reply:
Sounds good, thank you for your feedback!